modules/linear_boltzmann_solvers/a_lbs_solver/source_functions/source_function.cc

```diff
@@ -17,19 +17,19 @@ void
 SourceFunction::operator()(LBSGroupset& groupset,
                            std::vector<double>& destination_q,
                            const std::vector<double>& phi_local,
-                           SourceFlags source_flags)
+                           Source source)
 {
-  if (source_flags & NO_FLAGS_SET) return;
+  if (source.Empty()) return;
```

Comment:
This was an interesting line. `NO_FLAGS_SET` was `0` so `anything & 0 == false`, so this shortcut did not work. Maybe, earlier in the code `NO_FLAGS_SET` had a non-zero value... This way it should be better...